liquibase-core/src/main/java/liquibase/sqlgenerator/core/AlterSequenceGenerator.java

```diff
@@ -68,7 +68,9 @@ public ValidationErrors validate(AlterSequenceStatement alterSequenceStatement,
 
         if (statement.getOrdered() != null) {
             if (statement.getOrdered()) {
-                buffer.append(" ORDER");
+                buffer.append(" ORDER ");
+            } else {
```

Comment:
It appears that there is at least one platform (MS SQL Server) that does not allow an ORDER clause, which could cause a problem. Could be protected by statement.getOrdered() always returning null for certain platforms?

Reply:
Hi, thanks for review.

You're right, so the liquibase documentation is wrong (says that "mssql" supports this) in:

* https://www.liquibase.org/documentation/changes/create_sequence.html
* https://www.liquibase.org/documentation/changes/alter_sequence.html

But my fix is just on NOORDER. Alter the sequence to switch from ORDER to NOORDER like it's already done for CYCLE/NOCYCLE or CACHE/NOCACHE.